src/Airship/Internal/Decision.hs

```diff
@@ -696,7 +693,16 @@ o16 r = do
     req <- lift request
     if requestMethod req == HTTP.methodPut
         then o14 r
-        else o18 r
+        else o17 r
+
+o17 r@Resource{..} = do
+    trace "o17"
```

Comment:
Might be nice at some point to tweak [for-GET](https://github.com/for-GET/http-decision-diagram) and have the (slightly augmented) airship flow graph.